include/platform/pressureFlowSensor.h

```diff
@@ -10,40 +10,54 @@
 #ifndef INC_PLATFORM_PRESSURE_FLOW_SENSOR_H_
 #define INC_PLATFORM_PRESSURE_FLOW_SENSOR_H_

```

Comment:
Renamed to sensor.h, as this is the only sensor type involved in this model of gVent. Since the sensor also has pressure and temperature, sensor encompasses a bunch of embedded sensors within the single FS6122 sensor unit.